### ISSUE TITLE
Sign git commits using openkeychain as hw token

### ISSUE THREAD

**Issue by S2-** [open-keychain/open-keychain#2525]
With a Youbikey [it is possible to sign git commits](https://www.engineerbetter.com/blog/yubikey-signed-commits/) made on a desktop/laptop with the key stored on the youbikey.

It would be awesome if this could somehow work with openkeychain as well.

## Expected Behavior
- Connect your phone to a computer somehow (bluetooth, sane wifi network, or usb cable).
- On your computer create a `git commit -S -m 'a signed commit'`
- Some magic happens, and the key stored on openkeychain is used to sign the git commit

## Current Behavior
Nothing happens

## Possible Solution
I have no idea how this could be accomplished, or if it is even possible to do.
Did someone already think about this? Is my use case stupid?

**Comment by JohnScott623** [comment]
Why not just copy the key to the computer running Git?

**Comment by S2-** [comment]
Yes I could. But that way the private key would be on the computer running git, and I wanted to prevent that. With a youbikey the private key never leaves the hw token.
